corehq/apps/hqadmin/templates/hqadmin/email/error_email.html

```diff
@@ -0,0 +1,63 @@
+<html>
+<body>
+{% if details %}
+<h3>Details</h3>
+<table>
+    <tbody>
+        {% for key, value in details.items %}
+        <tr>
+            <td>{{ key }}</td>
+            <td>{{ value }}</td>
+        </tr>
+        {% endfor %}
+    </tbody>
+</table>
+{% endif %}
+
+{% for line in tb_list %}
+{% if forloop.counter0 == 0 %}
+<div>
+    <h3>{{ line }}</h3>
+</div>
+{% elif 'site-packages' in line %}
+<div>
+    <small>{{ line }}</small>
+</div>
+{% else %}
+<div>
+    <b>{{ line }}</b>
+    <br />
```

Comment:
super nitpicky, but I think the convention is to favor just `<br>` instead of `<br />`. http://stackoverflow.com/questions/1946426/html-5-is-it-br-br-or-br